t/22-dashboard.t

```diff
@@ -151,7 +151,7 @@ check_test_parent('collapsed');
 # links are correct
 my @urls = $get->tx->res->dom->find('h2 a, .row a')->map('attr', 'href')->each;
 for my $url (@urls) {
-    next if ($url =~ /^#/);
+    next if ($url =~ /^#/ || $url =~ /^.*\:\//);
```

Comment:
I guess a comment here would not harm

Reply:
If you want to avoid regexes, i think you can also (not tested)

``` perl
next if Mojo::URL->new($url)->scheme;
```

should cover also the cases with ```#```

Reply:
Oh yes, that would be even better than a comment